examples/data-transfer-mssql/main.py

```diff
@@ -72,30 +85,30 @@ def extract_sql(file: str):
 
 
 def transfer_data(source_db_cursor, dest_db_cursor, dest_db_conn):
-    ''' Extracts fruits data from source database and stores them in destination database. '''
-    print('Extracting fruits data from source database.')
-    source_db_cursor.execute('SELECT * FROM fruits')
+    ''' Extracts users data from source database and stores them in destination database. '''
+    print('Extracting users data from source database.')
+    source_db_cursor.execute('SELECT * FROM users')
     rows = source_db_cursor.fetchall()
 
-    print('Transferring fruits data to destination database.')
+    print('Transferring users data to destination database.')
     for row in rows:
-        dest_db_cursor.execute('INSERT INTO fruits VALUES (?, ?, ?)', (row.id, row.name, row.quantity))
+        dest_db_cursor.execute(SQL_INSERT_DATA, (row.id, row.name, row.city))
 
     print(f'{len(rows)} rows transferred\n')
     dest_db_conn.commit()
 
 
-def display_fruits(db_cursor):
-    ''' Displays fruits data. '''
-    db_cursor.execute('SELECT * FROM fruits')
+def display_users(db_cursor):
+    ''' Displays users data. '''
+    db_cursor.execute('SELECT * FROM users')
```

Comment:
This solves the ordering issue.
```suggestion
    db_cursor.execute('SELECT * FROM users ORDER BY id')
```

Reply:
Shouldn't this be the same as insert order?

Reply:
Yes, now it works as intended.